src/publisher.js

```diff
@@ -40,6 +39,10 @@ class Exchanges {
 
   async publisher({rootUrl, schemaset, client, sendDeadline, publish, aws}) {
     let publisher;
+    if (process.env.NODE_ENV ==='development') {
```

Comment:
Let's use `!== 'production'` here, to be the logical negative of the other conditoinal.  Also include the space :)

Reply:
why lint didn't catch that